data-reporting-service/organization-service/src/test/resources/orgtransformed/OrgReporting.json

```diff
@@ -0,0 +1,201 @@
+        "field": "add_user_id"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "general_comments"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "street_address_2"
+      },
+      {
+        "type": "int64",
+        "optional": false,
+        "field": "organization_uid"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "refresh_datetime"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "last_chg_user_name"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "quick_code"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "zip"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "country"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "city"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "phone_comments"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "fax"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "facility_id_auth"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "email"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "street_address_1"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "address_comments"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "last_chg_time"
+      },
+      {
+        "type": "string",
+        "optional": true,
+        "field": "facility_id"
+      }
+    ]
+  },
+  "payload": {
+    "organization_uid": 10036000,
+    "local_id": "ORG10000000AL01",
+    "record_status": "ACTIVE",
+    "general_comments": "1.0.5 PHA 8 Bobby H. Bryant-Area Admin. & Admin for Autauga County",
+    "entry_method": "N",
+    "stand_ind_class": "Health Care and Social Assistance",
+    "organization_name": "Autauga County Health Department",
+    "quick_code": "AUTCHD",
+    "facility_id": "A4646",
+    "facility_id_auth": "OTH",
+    "city": "Prattville",
+    "state": "Alabama",
+    "state_code": "01",
+    "zip": "36067-0000",
+    "county": "Autauga County",
+    "county_code": "01001",
+    "country": "840",
+    "address_comments": "Testing address Comments!",
+    "phone_work": "3343613743",
+    "phone_ext_work": "1234",
+    "email": "john.doe@test.com",
+    "phone_comments": "Testing phone Comments!",
+    "fax": "7072834657",
+    "add_user_id": null,
+    "add_user_name": null,
+    "add_time": "2004-07-22 10:48:53.343",
+    "last_chg_user_id": 10180119,
+    "last_chg_user_name": "ZRY4SFCLNAIUPX, CRPVQBYEAZNDJS6H",
+    "last_chg_time": "2024-04-05 18:55:54.177",
+    "refresh_datetime": null,
```

Comment:
Refresh_datetime is auto-generated and handled by sql server. This should removed from the payload.

Reply:
Will remove it.